docs/io/unified.rst

```diff
@@ -230,6 +230,130 @@ At this time, the ``meta`` attribute of the :class:`~astropy.table.Table` class
+
+Thus, a well defined protocol has been developed to allow the storage of mixin
+columns in FITS while still maintaining considerable ``round-tripping``.
+
+* Quantity
+
+  Since the only difference between `~astropy.table.Table` and
+  `~astropy.table.QTable` is that the former converts columns with an
+  associated ``unit`` attribute to `~astropy.units.Quantity`
+  objects, a Quantity object in a QTable is treated the same way as a
+  standard Column with a unit attribute in a Table. It used the ``TUNITn``
+  column keyword to incorporate the unit attribute of Quantity.  
+
+      >>> from astropy.table import QTable
+      >>> from astropy.units import Quantity
+      >>> t = QTable([Quantity([1, 2], unit='Angstrom')])
+      >>> t.write('my_table.fits', overwrite=True)
+      >>> qt = QTable.read('my_table.fits')
+      >>> qt
+      <QTable length=2>
+        col0
+      Angstrom
+      float64
+      --------
+           1.0
+           2.0
+
+All the other ``mixin columns`` being quite difficult to store
+in FITS tables, due to reasons including extensive metadata and
+no precise mapping to the FITS standard, are treated separately
+by using the argument ``astropy_native`` in the FITS read/write API.
+Thus, an Astropy Table or QTable can read/write these mixin columns
+from/to FITS tables. Currently this is limited to `~astropy.time.Time`
+columns since the FITS standard has been extended to include time.
+
+* Time
+
+  Time as a dimension in astronomical data presents challenges in its 
+  representation in FITS files. The standard has therefore been extended to
+  describe rigorously the time coordinate in the ``World Coordinate System``
+  framework. Refer to `FITS WCS paper IV <http://adsabs.harvard.edu/abs/2015A%26A...574A..36R/>`_
+  for details.
+
+  Allowing `~astropy.time.Time` columns to be written as normal columns in FITS
+  tables thus involves storing time values in a way to ensure retention
+  of precision and mapping the elaborate metadata associated with it to
+  the relevant FITS keywords.
+
+  In accordance with the standard which states that, in binary tables one
+  may use pairs of doubles, the Astropy Time column is written in such a
+  table as a vector of two doubles ``(TFORM n = ‘2D’)`` (jd1, jd2).
+  This reproduces the time values to double-double precision and is the
+  "lossless" version exploiting the higher precision provided in binary
+  tables. Considerable round-tripping of astropy written FITS binary tables
+  containing time columns has been achieved by mapping metadata,
+  ``scale`` and ``singular location`` to corresponding keywords.
+
+  This new functionality has been provided in both, the Table and QTable API,
+  using the optional argument ``astropy_native``. Users can
+  explicitly set it to `True` to allow the writing and reading of Time
+  to and from FITS without much loss of precision and metadata. By default,
+  this functionality is off and users can store the original time
```

Comment:
I'm not sure how to say that better, but I don't like "original time representation". Who knows what the original was? Maybe: just store the raw numeric values used by astropy time, e.g. two floating point numbers for (jd1, jd2) with no extra meta data in the header.